server/lib/accounts.coffee

```diff
@@ -19,6 +19,14 @@ Accounts.onCreateUser (options, user) ->
 	# console.log 'options ->',JSON.stringify options, null, '  '
 	# console.log 'user ->',JSON.stringify user, null, '  '
 
+	userEmail = user.emails[0].address;
```

Comment:
Should exists an setting to enable/disable the domain verification cuz there are installations that don't require email on signup